Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -105,12 +106,17 @@ var TouchableOpacity = React.createClass({
   },
 
   touchableHandleActivePressOut: function() {
-    this.setOpacityTo(1.0);
+    var childStyle = this.refs[CHILD_REF].props.style;
+
+    if (typeof childStyle === 'number') {
```

Comment:
You want to use `flattenStyle` here. The type of childStyle can be a number, or an object, or an array of number or objects or array... :)